sysutils/fluent-bit/Portfile

```diff
@@ -11,7 +11,7 @@ PortGroup           legacysupport               1.1
 
 legacysupport.newest_darwin_requires_legacy     15
 
-github.setup        fluent fluent-bit 2.2.2 v
+github.setup        fluent fluent-bit 3.0.4 v
```

Comment:
@herbygillot @l2dy most recent version appears to be 3.0.6 - perhaps update to that and see if the build failure on macOS-11 disappears? If not, one should probably still merge this PR - possibly restricting older systems to the 2.x branch; no point in keeping this PR hanging in the queue.

Reply:
> If not, one should probably still merge this PR - possibly restricting older systems to the 2.x branch; no point in keeping this PR hanging in the queue.

We could also set `known_fail` on older OSes, if branching or subport is not desired.